tests/foreman/cli/test_remoteexecution.py

```diff
@@ -516,6 +517,12 @@ def test_positive_run_receptor_installer(self):
         )
         assert 'project-receptor.satellite_receptor_installer' in result
         assert 'Exit status: 0' in result
+        # check that there is one receptor conf file and it's only readable by the receptor user and root
+        result = ssh.command(
+            'ls -l /etc/receptor/*/receptor.conf | grep -- \'-r--------\\. 1 foreman-proxy\' && ls -l /etc/receptor/*/receptor.conf | wc -l | grep \'^1$\'',
```

Comment:
This line is too long, Travis complains about it.

For permissions and ownership, it's better to use `stats` instead of `ls`:
```bash
$ stat --format '%a:%U' /etc/fstab 
644:root
```

How expensive is ssh.command, anyway? Could we break it down to two commands and assert on their output?
There are some helpers in robottelo to re-use SSH session, might help speeding things up if time is a concern.